src/xrCommon/xr_deque.h

```diff
@@ -1,6 +1,6 @@
 #pragma once
 #include <deque>
-#include "xrCore/xrMemory.h"
+#include "xrCore/Memory/XRayAllocator.h"
```

Comment:
Do you really want to rollback changes from 170484 here? So, maybe remove these changes directly in 170484 to keep the diffs more simple?

Reply:
Fixed